api/src/search/backend/load_agencies_to_index.py

```diff
@@ -0,0 +1,90 @@
+from enum import StrEnum
+from typing import Sequence
+
+from pydantic import Field
+from sqlalchemy import select
+from sqlalchemy.orm import selectinload
+
+from src.adapters import db, search
+from src.api.agencies_v1.agency_schema import AgencyResponseSchema
+from src.db.models.agency_models import Agency
+from src.task.task import Task, logger
+from src.util.datetime_util import get_now_us_eastern_datetime
+from src.util.env_config import PydanticBaseEnvConfig
+
+SCHEMA = AgencyResponseSchema()
+
+
+class LoadAgenciesToIndexConfig(PydanticBaseEnvConfig):
+    shard_count: int = Field(default=1)
+    replica_count: int = Field(default=1)
```

Comment:
For these, give them more specific names so we can set them in terraform. Something like `LOAD_AGENCY_SEARCH_SHARD_COUNT`

For context - these need to be 3 & 2 respectively in prod otherwise the index creation will fail.